src/payment/strategies/mollie/mollie-payment-strategy.ts

```diff
@@ -149,17 +162,22 @@ export default class MolliePaymentStrategy implements PaymentStrategy {
     }
 
     private _mountElements() {
+
         const mollieClient = this._getMollieClient();
-        const { cardNumberId, cardCvcId, cardExpiryId, cardHolderId, styles } = this._getInitializeOptions();
+        const { containerId, cardNumberId, cardCvcId, cardExpiryId, cardHolderId, styles } = this._getInitializeOptions();
+
+        const container = document.getElementById(containerId);
 
-        this._cardHolderElement = mollieClient.createComponent('cardHolder', { styles });
-        this._cardNumberElement = mollieClient.createComponent('cardNumber', { styles });
-        this._verificationCodeElement = mollieClient.createComponent('verificationCode', { styles });
-        this._expiryDateElement = mollieClient.createComponent('expiryDate', { styles });
+        if (container?.style.display !== 'none') {
```

Comment:
this behaviour might be worth documenting in the interface, what happens when containerId is provided vs when not